src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/FileApiInteropRestore.depproj

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <RestoreOutputPath>$(MSBuildThisFileDirectory)obj</RestoreOutputPath>
+    <OutputPath>$(BinDir)</OutputPath>
+    <Language>C#</Language>
+    <ContainsPackageReferences>true</ContainsPackageReferences>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
```

Comment:
yup, anything compatible with netstandard would have worked because of the way I structured the package, plus we don't really care since we don't have TFM-specific assets on it, only runtime/native ones.